### PR TITLE
Add sudo to template node

### PR DESCRIPTION
**Pull request by AurevoirXavier**
To make the template node work out of the box.

Many new developers have complained about not being able to perform a runtime upgrade.

Unfortunately, there is no sudo or democracy available for this task.

Therefore, I have added the sudo in this PR.

### CODE REVIEW

**Comment by paritytech-cicd-pr** [comment]
The CI pipeline was cancelled due to failure one of the required jobs.
**Job name**: test-linux-stable
**Logs**: https://gitlab.parity.io/parity/mirrors/cumulus/-/jobs/3004970

**Comment by gilescope** [comment]
Could you put the sudo pallet as 15 please? There's an extended parachain template that people are developing (not yet done, so not yet public), and they're using 15 for sudo. I think it's a good idea to add sudo - many parachains start off with it before migrating to more decentralised governance structures...

File: parachain-template/runtime/src/lib.rs [hunk 1]
```diff
@@ -485,6 +485,9 @@ construct_runtime!(
 		Balances: pallet_balances = 10,
 		TransactionPayment: pallet_transaction_payment = 11,
 
+		// Utility
```
Comment (gilescope) [inline parachain-template/runtime/src/lib.rs:488]:
```suggestion
		// Governance
```

**Review by gilescope — APPROVED** [review]

**Comment by gilescope** [comment]
Benchmarks are complaining about sudo but it has benchmarks these days... https://github.com/paritytech/substrate/pull/13880 so not sure why it's complaining?

**Review by ggwpez — APPROVED** [review]

**Comment by ggwpez** [comment]
> Benchmarks are complaining about sudo but it has benchmarks these days... [paritytech/substrate#13880](https://github.com/paritytech/substrate/pull/13880) so not sure why it's complaining?

Could be that the `runtime-benchmarks` feature is missing in the runtime Cargo.toml.

**Comment by gilescope** [comment]
@AurevoirXavier in `parachain-template/runtime/Cargo.toml` can you add `"pallet-sudo/runtime-benchmarks",` under the `runtime-benchmarks` feature please?

**Comment by gilescope** [comment]
bot merge

**Pull request merged**